flixel/group/FlxGroup.hx

```diff
@@ -107,6 +115,18 @@ class FlxTypedGroup<T:FlxBasic> extends FlxBasic
 	{
 		super.destroy();
 		
+		if (addedToGroup != null)
```

Comment:
The usual `FlxDestroyUtil` one-liner should work?

Reply:
I'm not seeing `destroy` in `FlxTypedSignal`'s code completion. It's defined in the real class but not the abstract.

Reply:
The abstract should forward it then.

Reply:
It does for the compiler, but not for code completion. Adding `@:forward` to `FlxTypedSignal` makes it show up, if it's needed.